src/strings/PowerFxResources.en-US.resx

```diff
@@ -4579,4 +4607,32 @@
     <value>Delegation warning. The result of this argument '{0}' may be truncated for large data sets before being passed to the '{1}' function.</value>
     <comment>Error message when an argument to non-delegable function has possible delegation and resulting rows may be truncated</comment>
   </data>
+  <data name="ErrNeedPrimitive" xml:space="preserve">
+    <value>Expected a text, numeric, boolean, or date/time value.</value>
+    <comment>Error Message.</comment>
```

Comment:
>Error Message. [](http://example.com/codeflow?start=13&length=14)

Please expand the comment to something like "Error message shown to the user when a function expects an expression that returns a primitive value but it doesn't."

Similar to the other error messages